app/src/processing/app/contrib/StatusPanel.java

```diff
@@ -235,9 +236,9 @@ public void update(ContributionPanel panel) {
       latestVersion = "Update";
     }
 
-    if(currentVersion != null){
-      currentVersion = "Version " + currentVersion;
-    }else{
+    if (currentVersion != null) {
+      currentVersion = "v" + currentVersion;
```

Comment:
This makes an assumption about the format of the version field that may not always be correct. Putting "v" in front won't make sense in some cases. Just leave it off.